Mods/DiscordRichPresence/Src/DiscordRpc.cpp

```diff
@@ -0,0 +1,203 @@
+#include "DiscordRpc.h"
+
+#include "Hooks.h"
+#include "Logging.h"
+#include "DiscordClient.h"
+
+#include <Glacier/ZScene.h>
+#include <regex>
+
+DiscordClient discordClient;
```

Comment:
Static globals should be avoid. This would probably be better as a member variable in the `DiscordRpc` plugin.